engine/src/flutter/lib/web_ui/lib/src/engine/semantics/semantics_helper.dart

```diff
@@ -79,6 +79,15 @@ abstract class SemanticsEnabler {
   /// Or if the received [DomEvent] is suitable/enough for enabling the
   /// semantics. See [tryEnableSemantics].
   bool shouldEnableSemantics(DomEvent event) {
+    // Simply tabbing into the placeholder element should not cause semantics
+    // to be enabled. The user should actually click on the placeholder.
+    if (event.isA<DomKeyboardEvent>()) {
+      event as DomKeyboardEvent;
+      if (event.key == 'Tab') {
+        return true;
+      }
```

Comment:
I don't understand this comment, it says "simply by tabbing should not cause semantics to be enabled" but then it returns true if event.key is 'Tab'?

Reply:
This looks like a fix for a different issue, please list it in the "Fixes" of the description of this PR?

Reply:
The name of the method - `shouldEnableSemantics` - is misleading. `true` return value means "yes, forward the event to the framework; do not consume it". I plan to do some renaming, but not in this PR. The reason I included it in this PR is because as I was testing focus functionality I noticed that stumbling upon the placeholder threw off focus management because it enabled semantics and completely changed the DOM structure. With this fix I can cleanly leave and re-enter the iframe that hosts the Flutter app.